### PR TITLE
fix(RELEASE-1061): linting issues in collect-gh-params task

### PR DESCRIPTION
**Pull request by mmalina**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -47,25 +47,16 @@ jobs:
...
-            echo "Checkton found issues"
-            exit 1
-          else
-            echo "Checkton found no issues"
-          fi
+          find-copies-harder: false
+          # Set to false to disable the differential check
+          differential: false
+          # Add a regex to include only the files you want to check
+          include-regex: "tasks/collect-gh-params/.*\\.yaml"
```
Comment (mmalina) [inline .github/workflows/lint.yaml]: Remove changes before merging.

**Review by johnbieren — COMMENTED** [review]
LGTM, will approve once workflow change is removed

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (mmalina) [reply .github/workflows/lint.yaml]: Removed. Ready to be merged.

**Comment by mmalina** [comment]
> LGTM, will approve once workflow change is removed

OK, thanks. Please check now.

**Review by johnbieren — APPROVED** [review]

**Review by happybhati — APPROVED** [review]

**Pull request merged**